### ISSUE TITLE
--no-metadata stopped working in v1.10.1

### ISSUE THREAD

**Issue by iownthegame** [caarlos0/svu#103]
Hi there, we have code logic below:

```shell
    next_version="$(svu n --force-patch-increment --no-metadata --tag-mode all-branches ${svu_extra_args[@]} 2> /dev/null)"
    if [[ -z "${next_version}" ]]; then
      next_version="$(svu n --force-patch-increment --no-metadata --tag-mode all-branches)"
    fi
```

We got an error about `unknown long flag '--no-metadata'`
<img width="561" alt="image" src="https://user-images.githubusercontent.com/1487369/224285870-7cea1e3a-3ada-417e-9d25-c9710bfe734e.png">

I wonder if it's intended to deprecate the flag, where can I find more information about the changelogs? thanks

**Comment by caarlos0** [comment]
Yes, it was removed. When forcing patch increments does the same thing. 

Check out the readme for examples. 

PS: always good to search open and closed issues before creating a new one. ☺️

Cheers!
